packages/es-components/src/components/containers/table/Table.js

```diff
@@ -1,32 +1,130 @@
-import styled from 'styled-components';
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
 
-const Table = styled.table`
+const TableBase = styled.table`
   border-collapse: collapse;
+  border: 0;
   border-spacing: 0;
+  margin: 0 0 25px 0;
+  padding: 0;
   width: 100%;
-`;
 
-const TableRow = styled.tr`
-  &:last-of-type td {
-    border: 0;
+  th {
+    border-bottom: 2px solid ${props => props.theme.colors.gray4};
+    font-weight: bold;
+    line-height: ${props => props.theme.sizes.baseLineHeight};
+    padding: ${props => props.cellPadding};
+    text-align: left;
+    vertical-align: bottom;
   }
-`;
 
-const TableBodyCell = styled.td`
-  border-bottom: 1px solid ${props => props.theme.colors.gray4};
-  padding: 8px;
-  text-align: left;
-`;
+  td {
+    border-top: 1px solid ${props => props.theme.colors.gray4};
+    line-height: ${props => props.theme.sizes.baseLineHeight};
+    padding: ${props => props.cellPadding};
+    vertical-align: top;
+  }
+
+  tbody {
+    ${props =>
+      props.hasStripes &&
+      css`
+        tr:nth-of-type(odd) {
+          background-color: ${props.theme.colors.gray0};
+        }
+      `};
+  }
+
+  && tbody {
+    ${props =>
+      props.hasHover &&
+      css`
+        tr:hover {
+          background-color: ${props.theme.colors.gray1};
+        }
+      `};
+  }
 
-const TableHeaderCell = styled.th`
-  border-bottom: 2px solid ${props => props.theme.colors.gray4};
-  padding: 8px;
-  text-align: left;
+  ${props =>
+    props.isResponsive &&
```

Comment:
any reason we need this vs just having this be default behavior?

Reply:
There may be cases where we just want the basic table on smaller screens for simple data.

Reply:
sounds good. should the default for that prop be set to true instead of false?